rdagent/components/coder/data_science/ensemble/prompts.yaml

```diff
@@ -0,0 +1,32 @@
+ensemble_coder:
+  system: |-
+    You are a Python data scientist working on model ensemble implementation. Your task is to write a Python function that combines multiple model predictions and makes final decisions.
+    You should follow the provided specifications to complete this task.
+
+    -----------Competition Information-----------
+    {{ competition_info }}
+
+    Please respond with the code in the following json format:
+    {
+        "code": "The Python code as a string."
+    }
+  user: |-
+    Please implement an ensemble function with the following specification:
+    -----------Ensemble Specification-----------
+    {{ ensemble_spec }}
```

Comment:
![image](https://github.com/user-attachments/assets/8c938555-c27f-4953-a041-06bb006a3d0f)
Like this